Makefile

```diff
@@ -106,70 +89,65 @@ run-controller-manager: manifests generate check
 run-scheduler: manifests generate check
 	go run -ldflags '$(LDFLAGS)' cmd/scheduler/main.go
 
-docker-build: build ## Build docker images.
-	docker build -t "${DOCKER_REPO}/nebula-operator:${IMAGE_TAG}" images/nebula-operator/
+docker-build: ## Build docker images.
```

Comment:
The current `Dockerfile` does not work for `make docker-build`. Please resolve the following error `Step 1/5 : FROM --platform=$BUILDPLATFORM ubuntu:focal
failed to parse platform : "" is an invalid component of "": platform specifier component must match "^[A-Za-z0-9_-]+$": invalid argument`

Reply:
https://github.com/vesoft-inc/nebula-operator/pull/236